CHANGELOG.md

```diff
@@ -11,6 +11,30 @@ _The old changelog can be found in the `release-2.6` branch_
 
 # Changes Since v3.2.1
 
+# v3.3.0 - [2019.06.17]
+
+## Changed defaults / behaviors
+
+  - Remote login and status commands will now use the default remote if a remote name is not supplied
+  - Added Singularity hub (`shub`) cache support when using the `pull` command
+  - Clean cache in a safer way by only deleting subdirectories
+  - Improvements to the `cache clean` command 
```

Comment:
Dosnt this line mean the same as the line right above it?

Reply:
No.  Didn't you add shub caching and some other improvements?  I didn't know what all you changed so I just wrote this as a catch all.

Reply:
Uh yeah, right! Its fine as it is